### PR TITLE
add uncheck service active in restart func

### PR DESCRIPTION
**Pull request by nanli1**
Sometimes the service cannot be starting after we updated config file

### CODE REVIEW

**Review by chloerh — APPROVED** [review]

File: virttest/utils_libvirtd.py [hunk 1]
```diff
@@ -119,14 +119,15 @@ def stop(self):
             result.append(daem_item.stop())
         return all(result)
 
-    def restart(self, reset_failed=True):
+    def restart(self, reset_failed=True, wait_for_start=True):
```
Comment (dzhengfy) [inline virttest/utils_libvirtd.py:122]: Could you also include start() for this enhancement?

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: virttest/utils_libvirtd.py (hunk 1 above)
Comment (nanli1) [reply virttest/utils_libvirtd.py:124]: Thanks dzheng, Done

File: virttest/utils_libvirtd.py [hunk 2]
```diff
@@ -103,13 +103,15 @@ def _check_start():
...
-    def start(self, reset_failed=True):
+    def start(self, reset_failed=True, wait_for_start=True):
         result = []
         for daem_item in self.daemons:
             if reset_failed:
                 daem_item.reset_failed()
             if not daem_item.start():
                 return False
+            if wait_for_start:
+                result.append(self._wait_for_start())
```
Comment (cliping) [inline virttest/utils_libvirtd.py:114]: Why _wait_for_start() twice?

Comment (nanli1) [reply virttest/utils_libvirtd.py:114]: Thanks liping ,DONE

**Review by cliping — APPROVED** [review]
LGTM

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**